INSTALL.rst.txt

```diff
@@ -14,24 +14,18 @@ Prerequisites
 
 Building NumPy requires the following installed software:
 
-1) For Python 3, Python__ 3.5.x or newer.
+1) Python__ 3.6.x or newer.
 
-   On Debian and derivative (Ubuntu): python python-dev
+   Please note that the Python development headers also need to be installed.
+   This is not always the case, e.g. on Debian/Ubuntu one needs to install two
+   packages: `python` and `python-dev`.
+   On Windows and macOS this is normally not an issue.
```

Comment:
Maybe ".. also need to be installed, e.g., on Debian/Ubuntu one needs to install both `python3` and `python3-dev`."

Reply:
thanks, updated to that

Reply:
I suggested something different. "This" may not be clear to non-native English speakers who may think you mean that the headers themselves do not always need to be installed.

Reply:
updated again